.gitignore

```diff
@@ -1,3 +1,11 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
hi @ralflukner , these seem like they are specific to only your setup and so should probably be removed.